src/components/AvatarGroup/AvatarGroupCounter.jsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import Tooltip from "../Tooltip/Tooltip";
+import Avatar from "../Avatar/Avatar";
+import PropTypes from "prop-types";
+import AvatarGroupCounterTooltipContainer from "./AvatarGroupCounterTooltipContainer";
+import cx from "classnames";
+import Counter from "../Counter/Counter";
+import styles from "./AvatarGroupCounter.module.scss";
+
+const AvatarGroupCounter = ({
+  counterTooltipAvatars,
+  counterProps,
+  counterTooltipCustomProps,
+  avatarSize,
+  avatarType
+}) => {
+  const {
+    color: counterColor = Counter.colors.LIGHT,
+    count: counterValue = counterTooltipAvatars.length,
+    prefix: counterPrefix = "+",
+    maxDigits: counterMaxDigits = 3
+  } = counterProps || {};
+
+  const counterSizeStyle = styles[avatarSize?.toString()];
+  const counterColorStyle = styles[counterColor];
+
+  if (!counterTooltipAvatars.length && !counterValue) {
```

Comment:
```suggestion
  if (counterTooltipAvatars === undefined || counterTooltipAvatars.length === 0) {
```

counterValue is just the length of counterTooltipAvatars or I did not understand?

Reply:
counterValue = count || counterTooltipAvatars.length
where `count` is prop passed by user - value to be displayed in `Counter`